examples/06_manipulating_images/plot_nifti_labels_simple.py

```diff
@@ -0,0 +1,61 @@
+"""
+Simple example of NiftiLabelsMasker use
```

Comment:
This is technically right, but I think that we should strive to label and introduce examples with a user-oriented perspective: what this shows is how to check the regions provided by an atlas, and then the correspondence with a user-supplied image.